README.md

```diff
@@ -19,6 +19,8 @@ at close-to-FFT speeds;
 under the hood, this involves detailed kernel design, custom spreading/interpolation stages, and plain FFTs performed by cuFFT.
 See the [documentation for FINUFFT][3] for a full mathematical description of the transforms and their applications to signal processing, imaging, and scientific computing.
 
+**Note**: We are currently in the process of adapting the cuFINUFFT interface to closer match that of FINUFFT. This will likely break code depending on the current interface once the next release is published. At this point we will publish a migration guide that will detail the exact changes to the interfaces.
```

Comment:
I think we should also say that this is the last release of cufinufft as its own repo, and future access to cufinufft will be as part of the main finufft repo.

Reply:
Yeah I'll put that in the main readme (this one's only for the Python package).

Reply:
Done (see 21cef9d53428b3b7359660d53737fc6881be730a).